sdccc/src/test/java/com/draeger/medical/sdccc/messages/TestMessageStorage.java

```diff
@@ -1820,10 +1827,138 @@ void testDetermineCharsetFromMessageFromHttpHeader(@TempDir final File dir) thro
         }
     }
 
+    /**
+     * Ensures that the test run is not invalidated when an encoding error is detected
+     * but SummarizeMessageEncodingErrors is set to true in the configuration.
```

Comment:
The documentation no longer matches. Test run is invalidated when an encoding error is detected.

Reply:
Not any more. Now the description matches again.